### ISSUE TITLE
.card-block missing clearfix

### ISSUE THREAD

**Issue by aimeos** [twbs/bootstrap#18583]
.card components that contain floats (e.g. "col-sm-6" grids) have no height because of the floating elements. Can be fixed by adding:
```
.card-block:after {
	display: block;
	clear: both;
	content: "";
}
```

**Comment by cvrebert** [comment]
Erm, you're referring to clearfix, right?

**Comment by aimeos** [comment]
Not the .clearfix class itself. Divs with .card-block have simply no height if they contain only floating elements like grids.

**Comment by cvrebert** [comment]
> Not the .clearfix class itself.

I was referring to the abstract concept of clearfix. Bootstrap implements it both as a mixin and as a class though.

> Divs with .card-block have simply no height if they contain only floating elements like grids.

Yes, that's what clearfix fixes. So the obvious (and seemingly reasonable) solution here would be to make `.card-block` include the `clearfix` mixin.

**Comment by aimeos** [comment]
Yes, sure.

**Comment by cvrebert** [comment]
Fixed by #18655.
